spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -2668,6 +2673,12 @@ def setup_context_menu(self):
             self, _("Zoom reset"), shortcut=QKeySequence("Ctrl+0"),
             triggered=self.zoom_reset.emit)
 
+        # Docstring
+        writer = self.writer_docstring
+        self.docstring_action = create_action(
+            self, _("Write Docstring"),

```

Comment:
I suggest ``Generate function docstring`` as clearer UI text (also note our general convention is Sentance case, not Title Case, though we're not 100% consistent about it).